Makefile

```diff
@@ -95,7 +96,7 @@ tree-status: ## Verify tree is clean and all changes are committed
 build-cron: build-pubsub build-bq-transfer build-github-server build-webhook build-add-script \
 	  build-validate-script build-update-script
 
-build-targets = generate-docs build-proto build-scorecard build-cron dockerbuild
```

Comment:
Why are we removing `dockerbuild`? Don't we want both `dockerbuild` and `ko-build`?

Reply:
Why do we want dockerbuild and ko build? The progression is to replace dockerfiles with `ko`

Reply:
Before we remove `dockerbuild`, have we confirmed that the images generated by `ko` vs. `docker` don't diverge? If you have tested this locally somehow, do paste the results in the PR description.

Reply:
To clarify what I mean, `ko-build-batch-controller` for example, does not include the data files `cron/data/projects.*csv` which are required for it to function correctly. There might be other such nuances and subtleties which we might miss by completely doing away with `dockerbuild` without thorough testing.

We can submit this PR as is, but let's keep the `dockerbuild` target.

Reply:
Agree, it is a valid argument. I will also include the docker build.